django/contrib/admin/templates/admin/login.html

```diff
@@ -58,7 +58,7 @@
   </div>
   {% endif %}
   <div class="submit-row">
-    <label>&nbsp;</label><input type="submit" value="{% translate 'Log in' %}">
+    <input type="submit" value="{% translate 'Log in' %}">
```

Comment:
This affects the layout by a small number of pixels. **But** the button isn't centered perfectly anyway... — so it's OK I think. 

(Might be a place where flexbox would be a better choice now, than a hard-coded padding value.)

Reply:
Do you know if there is any reason this element isn't styled as `align: text-center`? Is that a RTL issue? If there is no issue, I can look at making that change so it is centered.

Reply:
I don't know, no. It had always appeared centred, but when I got the ruler out... :)

Reply:
I went down a bit of a rabbit hole cleaning up and simplifying the admin CSS.

The fix to center the button is in #12847.